store/store.go

```diff
@@ -1519,7 +1528,13 @@ var download = func(ctx context.Context, name, sha3_384, downloadURL string, use
 		dlSize = float64(resp.ContentLength)
 		pbar.Start(name, dlSize)
 		mw := io.MultiWriter(w, h, pbar)
-		_, finalErr = io.Copy(mw, resp.Body)
+		var limiter io.Reader
+		limiter = resp.Body
+		if limit := dlOpts.RateLimit; limit > 0 {
+			bucket := ratelimit.NewBucketWithRate(float64(limit), 2*limit)
```

Comment:
this if has no coverage :-/

Reply:
Thanks, indeed, I added this (and some more) tests now.